torch/nested/_internal/sdpa.py

```diff
@@ -604,7 +604,7 @@ def _pad_last_dim(
 
 # TODO: coalesce with torch/nn/utils/attention.py
 def _calculate_scale(query, scale):
-    softmax_scale = scale if scale is not None else math.sqrt(1.0 / query.size(-1))
+    softmax_scale = scale if scale is not None else torch.sym_sqrt(1.0 / query.size(-1))
```

Comment:
I really don't have time to figure out why tf Dynamo didn't properly translate `math.sqrt()` for this case while it does perfectly fine for a minimal example.